capa/render/result_document.py

```diff
@@ -490,7 +490,7 @@ def from_capa(cls, rule: capa.rules.Rule) -> "RuleMetadata":
             examples=rule.meta.get("examples", []),
             description=rule.meta.get("description", ""),
             lib=rule.meta.get("lib", False),
-            capa_subscope=rule.meta.get("capa/subscope", False),
+            is_subscope_rule=rule.meta.get("capa/subscope", False),
```

Comment:
i think this is a bug: we're using a name here that doesn't correspond with a property declared on the model.